content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -45,52 +45,31 @@ check the documentation for that version.
 {{< /note >}}
 
 <!-- body -->
-## Install and configure prerequisites
```

Comment:
We should keep a prerequisites heading here. Network configuration can be a subheading.

Reply:
OK. (I'd removed it because Network Configuration is now the _only_ subheading.)